sympy/series/limitseq.py

```diff
@@ -242,5 +242,5 @@ def limit_seq(expr, n=None, trials=5):
         # Maybe the absolute value is easier to deal with (though not if
         # it has a Sum). If it tends to 0, the limit is 0.
         elif not expr.has(Sum):
-            if _limit_seq(Abs(expr.xreplace({n: n_})), n_, trials).is_zero:
+            if _limit_seq(Abs(expr.xreplace({n: n_})), n_, trials) is not None and _limit_seq(Abs(expr.xreplace({n: n_})), n_, trials).is_zero:
```

Comment:
`_limit_seq` should not be computed twice. Do something like this instead:

        lim = _limit_seq(Abs(expr.xreplace({n: n_})), n_, trials)
        if lim is not None and lim.is_zero:

Reply:
Okay will make the changes.